### DISCUSSION TITLE
Alternative writing style for ingredients for listing

### DISCUSSION THREAD

**Discussion by MiK546** [cooklang/spec#85] [Ideas]
The spec description implies that the spec aims to target English only ("plain-english instructions"), but I'll throw this idea in as I think the spec easily fits any language. Currently the ingredients are listed in the recipe ingredient and grocery lists exactly as they are written in the instruction text. This is fine for languages like English where words don't have forms, but for example Finnish where words have forms this results in a weird sounding ingredient list. For example:

```
...sekoita ainekset @maidon{2%dl} kanssa.
```

In English this would be
```
...mix the ingredients with @milk{2%dl}.
```

But putting "maidon" (base form or this would be "maito" == "milk") to the ingredient list would be the same as listing "with milk"  in the list in English. In Finnish I would like the ingredient list to list "maito 2dl". Maybe the spec could be amended with an optional, separate base form for the ingredient for listing. Something like:

```
...sekoita ainekset @maidon|maito{2%dl} kanssa.
```

Which would result in the output being something like

```
maito 2dl
...
...sekoita ainekset maidon kanssa.
```

**Comment by dubadub** [comment]
Hi @MiK546, it's an absolutely valid problem you mentioned and the first priority to implement for us support of non-English languages. Here's a proposal I'm working on https://github.com/cooklang/spec/blob/non-english/proposals/0001-support-recipes-in-any-language.md. When I finish I'll share it for discussion.
